### PR TITLE
review tab

### PR DESCRIPTION
**Pull request by quannb99**
## Related Tickets:
- [#29305](https://edu-redmine.sun-asterisk.vn/issues/29305)
- [#29344](https://edu-redmine.sun-asterisk.vn/issues/29344)
## WHAT this PR does?
- Review tab
- Update review
- Delete review

## Notes Impacted Areas:
- N/A.

## Attachments:
- 
editreview
deleteReview

### CODE REVIEW

**Comment by quannb99** [comment]
ready

File: database/migrations/2020_10_04_160444_edit_reviews_table.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
```
Comment (huyenttm-0901) [inline database/migrations/2020_10_04_160444_edit_reviews_table.php]: 2 bảng khác nhau này em

**Comment by quannb99** [comment]
ready

File: app/Http/Controllers/BookController.php [hunk 2]
```diff
@@ -100,7 +104,9 @@ public function show($id)
             'randomBook',
             'categories',
             'rated',
-            'reviews']));
+            'reviews',
+            'reviewed',
+            ]));
```
Comment (thaonp104) [inline app/Http/Controllers/BookController.php:109]: indent 4 spaces

**Pull request merged**